components/tool.php

```diff
@@ -45,35 +45,6 @@ public function repair( $args, $assoc_args ) {
 			WP_CLI::error( $result[1] );
 		}
 	}
-
-	/**
-	 * Reinstall BuddyPress default emails.
-	 *
-	 * ## OPTIONS
-	 *
-	 * [--yes]
-	 * : Answer yes to the confirmation message.
-	 *
-	 * ## EXAMPLE
-	 *
-	 *     $ wp bp tool reinstall_emails --yes
-	 *     Success: Emails have been successfully reinstalled.
-	 */
-	public function reinstall_emails( $args, $assoc_args ) {
-		WP_CLI::confirm( 'Are you sure you want to reinstall BuddyPress emails?', $assoc_args );
-
-		$result = bp_admin_reinstall_emails();
-
-		if ( 0 === $result[0] ) {
-			WP_CLI::success( $result[1] );
-		} else {
-			WP_CLI::error( $result[1] );
-		}
-	}
 }
 
-WP_CLI::add_command( 'bp tool', 'BPCLI_Tool', array(
```

Comment:
You can keep this, `before_invoke`, as it is necessary for the `repair()` commands.